prometheus-to-sd/translator/translator.go

```diff
@@ -375,6 +377,11 @@ func convertToDistributionValue(h *dto.Histogram) *v3.Distribution {
 		prevVal = b.GetCumulativeCount()
 	}
 
+	// +Inf Bucket is implicit so it needs to be added
+	if !infSeen && count > int64(prevVal) {
```

Comment:
looks like this is addressing a different issue, could you add some background in the PR description?

Reply:
it's not a different issue because the +inf bucket is omitted in protobuf encoding but text encoding handles that specially (see https://github.com/prometheus/common/blob/main/expfmt/text_create.go#L232 if interested). However, it makes sense to separate the changes with description, for which I split the changes into 2 commits to make it clear.

Reply:
Thanks!  LGTM-)